client/src/views/AllocationView/AllocationView.tsx

```diff
@@ -171,8 +171,7 @@ const AllocationView = (): ReactElement => {
       isFetchingUserAllocation ||
       !isRewardsForProposalsSet ||
       currentEpoch === undefined ||
-      (isConnected && !userAllocations && isDecisionWindowOpen && currentEpoch > 1) ||
-      !rewardsForProposalsNew
```

Comment:
That looks suspicious. I will dig deeper why it was here, maybe truly without reason.